challenges/objects-arrays.js

```diff
@@ -97,29 +235,42 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
-console.log(lowerCase); 
+// const lowerCase = [];
+// zooAnimals.map(function(animal) {
+//  lowerCase.push(animal.animal_name.toLowerCase());
```

Comment:
You can do something like 

```javascript
 let lowerCase = zooAnimals.map(function (animal) {
  return(animal.animal_name.toLowerCase())
}
```

You don't need to create a new array and push in items. Map by default returns an array.